cmd/protoc-gen-go-tetragon/eventchecker/eventchecker.go

```diff
@@ -11,54 +11,54 @@ import (
 )
 
 // Generate generates boilerplate code for the eventcheckers
-func Generate(gen *protogen.Plugin, f *protogen.File) error {
-	g := common.NewCodegenFile(gen, f, "eventchecker")
-	yaml := common.NewCodegenFile(gen, f, "eventchecker/yaml")
+func Generate(gen *protogen.Plugin, files []*protogen.File) error {
+	g := common.NewCodegenFile(gen, files[0], "eventchecker")
+	yaml := common.NewCodegenFile(gen, files[0], "eventchecker/yaml")
```

Comment:
What's the logic behind `files[0]` here? How do we know that's the file we're looking for? A comment might be helpful.

Reply:
Its only used to generate the name of the new file. We take the Prefix of the filename and then use it for our generated files. It doesn't actually matter because all gen files are in the tetragon pkg. So its arbitrary I just picked any file.

For example if we find foobar.proto as the first file then we will generate foobar.pb.go files.